src/app/model/result/reply.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Where is this example used?

Reply:
Never. It was just handy to have an example. However, I delete it.